docs/learn.md

```diff
@@ -479,21 +461,15 @@ System.get("jquery");
 System.set("jquery", Module({$: $})); // WARNING: not yet finalized
 ```
 
-<blockquote class="alert alert--warning">
-  <h4>Additional polyfill needed</h4>
-  <p>
-    Since Babel defaults to using common.js modules, it does not include the
-    polyfill for the module loader API. Get it <a href="https://github.com/ModuleLoader/es6-module-loader">here</a>.
-  </p>
-</blockquote>
+:::caution Additional polyfill needed
+Since Babel defaults to using common.js modules, it does not include the
+polyfill for the module loader API. Get it <a href="https://github.com/ModuleLoader/es6-module-loader">here</a>.
+:::
 
-<blockquote class="alert alert--info">
-  <h4>Using Module Loader</h4>
-  <p>
-    In order to use this, you'll need to tell Babel to use the
-    <code>system</code> module formatter. Also be sure to check out <a href="https://github.com/systemjs/systemjs">System.js</a>.
-  </p>
-</blockquote>
+:::info Using Module Loader
+In order to use this, you'll need to tell Babel to use the
+<code>system</code> module formatter. Also be sure to check out <a href="https://github.com/systemjs/systemjs">System.js</a>.
```

Comment:
Can you remove the line break before `<code>`? Docusaurus will generate a `</p>` here.